README.md

```diff
@@ -49,6 +49,8 @@ Type: `string`
 
 If `'line'` is specified, `sourceMappingURL` will be written in a single-line comment (`//`).
 
+If `'hidden'` is specified, `sourceMappingURL` will not be written to support hidden sourcemaps.
```

Comment:
"hidden" sourcemaps is not really a thing I think. You could say something like:

_If 'hidden' is specified, no reference to the source map is included in the source file._

Reply:
I think `none` might be a better value then `hidden`. Then it says we have no(ne) comments. Otherwise it says we have hidden comments which is a bit odd.